src/configs/unary-elementwise-config.c

```diff
@@ -1680,10 +1680,11 @@ static void init_f32_to_qs8_cvt_config(void) {
       f32_to_qs8_cvt_config.init.f32_qs8_cvt = xnn_init_f32_qs8_cvt_scalar_fmagic_params;
       f32_to_qs8_cvt_config.element_tile = 4;
     }
-  #elif XNN_ARCH_RISCV
-    f32_to_qs8_cvt_config.ukernel = (xnn_vunary_ukernel_fn) xnn_f32_qs8_vcvt_ukernel__scalar_lrintf_u4;
-    f32_to_qs8_cvt_config.init.f32_qs8_cvt = xnn_init_f32_qs8_cvt_scalar_lrintf_params;
-    f32_to_qs8_cvt_config.element_tile = 4;
+  #elif XNN_ARCH_RISCV && XNN_ENABLE_RISCV_VECTOR
+    const struct xnn_hardware_config* hardware_config = xnn_init_hardware_config();
+    f32_to_qs8_cvt_config.ukernel = (xnn_vunary_ukernel_fn) xnn_f32_qs8_vcvt_ukernel__rvv_u8v;
```

Comment:
consider u4v or u2v, as we typically dont see u1v or u8v producing the fastest code.
m8 can cause the instructions to become bottlenecked on a large/slow instruction and/or register pressure.